apiary.apib

```diff
@@ -1801,6 +1801,36 @@ Get a list of all activity types.
                 + id: `9feb5f61-c6dd-4f9d-9b37-2813ada4e343` (string)
                 + name (string)
 
+# Group Todos
+
+## Todos [/todos]
+
+*Required scopes: `planning`*
+
+### todos.info [GET /todos.info]
+
+Get details for a single todo.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `879770c5-8a7f-0067-8c25-b4993bdb5163` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
```

Comment:
Q: Is the array necessary? Only one Todo can be returned right?

Reply:
Looking at other .info end points I'd say you're right. No need for data (array)